indexer/src/types/transaction.test.ts

```diff
@@ -490,3 +493,14 @@ Deno.test("toTypedEthTx EIP2930 Transaction before release with 31 bytes chunks
   assertEquals(ethTx.data, tx.data);
   assertEquals(ethTx.accessList, tx.accessList);
 });
+
+Deno.test("toTypedEthTx with real data", async () => {
+  const transactionsList = transactionsData.transactionsList;
+  for (let i = 0; i < transactionsList.length; i++) {
+    const transactions = transactionsList[i];
+    const ethTx = toTypedEthTx({
+      transaction: transactions[0].transaction,
+    }) as LegacyTransaction;
+    assertExists(ethTx);
```

Comment:
same here, I think we should have an issue to update this and assert on some `expected_output` json.

Reply:
https://github.com/kkrt-labs/kakarot-rpc/issues/1258

Reply:
Ok, thanks, I had just created #1259 but I will close it